lib/players/human.rb

```diff
@@ -0,0 +1,10 @@
+module Players
+	class Human < Player
+
+		def move(board)
+			puts "Please enter a number between 1 and 9"
+			input = gets.chomp
+		end
```

Comment:
No need to create a variable `input` here - implicit return is enough, just `gets.chomp` for the last line in the method.